WindowsFormsApp1/obj/Debug/friendcognition.csproj.FileListAbsolute.txt

```diff
@@ -66,3 +66,4 @@ C:\Users\almyk\Documents\GitHub\kaspegifs\fRiEndcognition\fRiEndcognition\Window
 C:\Coding\Projects\fRiEndcognition\WindowsFormsApp1\obj\Debug\friendcognition.LOGIN.resources
 =======
 >>>>>>> 85987dd... Fixed the Spaghetti
```

Comment:
bet as nesuprantu as idejau tuos failus i gitignora o cia vis tiek kazkas keicias ir dar rodo likusi >>>>>>> 85987dd... Fixed the Spaghetti, reik kazkaip ta sutvarkyt